docs/release_notes.rst

```diff
@@ -1,23 +1,11 @@
 .. _release-notes:
 
-=============
-Release Notes
-=============
-
-This page contains the release notes for Qiskit, starting from Qiskit 0.45, the first time that Qiskit and Qiskit Terra had the same versions.
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+Qiskit |version| release notes
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
 ..
-    These release notes get converted into Markdown files via the infrastructure at https://github.com/Qiskit/documentation, which then gets deployed to https://docs.quantum.ibm.com/api/qiskit/release-notes. Changes to these release notes will update those release notes the next time the API docs are generated.
-
-    To change release notes prior to Qiskit 0.45, update the Qiskit/documentation repository directly.
-
-.. release-notes::
-   :earliest-version: 0.45.0rc1
-
-.. release-notes::
-   :earliest-version: 0.45.0
-   :branch: stable/0.46
+    These release notes get converted into Markdown files via the infrastructure at https://github.com/Qiskit/documentation, which then gets deployed to https://docs.quantum.ibm.com/api/qiskit/release-notes. Changes to these release notes will update those release notes the next time the API docs are generated for this version.
 
 .. release-notes::
-   :earliest-version: 0.45.0
-   :branch: stable/0.45
+   :earliest-version: 1.0.0.0rc1
```

Comment:
I couldn't think of a way to automate this being "all the release notes since the last rc1 that started off a new minor series". I think this would need to be manually updated. 

It doesn't seem like a big deal if we forget to update it, though, since these dev release notes will never hit production. What matters is this file being set up correctly for releases.

Reply:
Yeah, minor releases are infrequent enough that it's probably not a big deal to leave it as a manual step.